.env

```diff
@@ -33,6 +33,18 @@ CONFIG_IMAGE_NAME=bahmni/default-config
 CONFIG_IMAGE_TAG=latest
 CONFIG_VOLUME=bahmni-config
 
+# Metabase Environment variables
+METABASE_IMAGE_TAG=v0.45.0-RC2
+METABASE_DB_NAME=metabase
+METABASE_DB_USER=metabase-user
+METABASE_DB_PASSWORD=password
+METABASE_DB_HOST=metabasedb
+METABASE_DB_PORT=5432
+
+#Metabase Postgres Environment Variables
+METABASE_POSTGRES_IMAGE_TAG=latest
```

Comment:
Please fix this also to a specific postgres DB image. Not to latest. Latest is risky and can be unstable. We want all users who use Metabase to get the exact same version of postgres DB (and not latest. which will depend on when you pulled the images).